app/lib/forms/sign_in.rb

```diff
@@ -30,5 +32,9 @@ def after_save
         ConfirmEmailMailer.confirmation_code_mail(to: email, code:).deliver_now
       end
     end
+
+    def skip_sending_otp
+      Rails.env.development? && @email == "admin@example.com"
```

Comment:
Realise this isnt ideal and maybe we can find a better way but thought simple enough for now